scripts/dockerlib.sh

```diff
@@ -44,6 +45,9 @@ LOG_FILE=$LOG_DIR/docker-handler.log
 CONFIG_FILE=$(ls $CONFIG_DIR | grep -E ^[0-9]+.settings$ | sort -n | tail -n 1)
 STATUS_FILE=$(echo $CONFIG_FILE | sed s/settings/status/)
 
+CONFIG=$CONFIG_DIR/$CONFIG_FILE
```

Comment:
Sure.  I was thinking CONFIG_FULL_PATH or something.  It's original name was as it is now, only lower case.  I'll change both of them to your suggestion.